staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -183,24 +183,18 @@ type Interface interface {
 	// and send it in an "ADDED" event, before watch starts.
 	Watch(ctx context.Context, key string, opts ListOptions) (watch.Interface, error)
 
-	// Get unmarshals json found at key into objPtr. On a not found error, will either
+	// Get unmarshals object found at key into objPtr. On a not found error, will either
 	// return a zero object of the requested type, or an error, depending on 'opts.ignoreNotFound'.
 	// Treats empty responses and nil response nodes exactly like a not found error.
 	// The returned contents may be delayed, but it is guaranteed that they will
 	// match 'opts.ResourceVersion' according 'opts.ResourceVersionMatch'.
 	Get(ctx context.Context, key string, opts GetOptions, objPtr runtime.Object) error
 
-	// GetToList unmarshals json found at key and opaque it into *List api object
-	// (an object that satisfies the runtime.IsList definition).
+	// GetList unmarshalls objects found at key range and opaque them into *List api
+	// object (an object that satisfies runtime.IsList definition).
```

Comment:
Add something like "If opts.Recursive is false, 'key' is used as an exact match. If opts.Recursive is true, 'key' is used as a prefix."?